spec/mailers/notification_mailer_spec.rb

```diff
@@ -7,9 +7,9 @@
     let(:mail) { NotificationMailer.notification_email(article) }
```

Comment:
Do you have a unit spec to check that emails get sent when an article is created?

You will also need to mock out that Mailer in the specs where you create an article and don't want it sending an email